resources/util.ts

```diff
@@ -459,6 +459,26 @@ const Util = {
     clearCombatantsOverride = clearFunc;
   },
   gameLogCodes: gameLogCodes,
+  shortName: (
+    name: string | undefined,
+    playerNicks: { [name: string]: string },
+  ): string => {
+    // TODO: make this unique among the party in case of first name collisions.
+    // TODO: probably this should be a general cactbot utility.
```

Comment:
I think this second `TODO` comment can be removed? Or at least replaced with a "we should clean up other instances to use this instead"?

Reply:
I think all other instances are using it, so I'll remove this comment.